soh/src/overlays/actors/ovl_En_Ge1/z_en_ge1.c

```diff
@@ -93,6 +94,18 @@ void EnGe1_Init(Actor* thisx, GlobalContext* globalCtx) {
     s32 pad;
     EnGe1* this = (EnGe1*)thisx;
 
+    // When spawning the gate operator, also spawn an extra gate operator on the wasteland side
+    if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_SHUFFLE_GERUDO_MEMBERSHIP_CARD) &&
+        Randomizer_GetSettingValue(RSK_SHUFFLE_OVERWORLD_ENTRANCES) && (this->actor.params & 0xFF) == GE1_TYPE_GATE_OPERATOR) {
+        Actor_Spawn(&globalCtx->actorCtx, globalCtx, ACTOR_EN_GE1, -1358.0f, 88.0f, -3018.0f, 0, 0x95B0, 0, 0x0302);
```

Comment:
Would be good to either explain or use an expression to show how you went from `0xFF` (used below) to `0x0302`

Reply:
This came from 3ds rando, I don't know where the `0x0300` came from but `0x0002` is from the custom guard type added, I can use an OR to at least make that clear.